src/Plugin/SecurityChecker.php

```diff
@@ -64,8 +65,12 @@ public static function canExecuteOnStage($stage, Build $build)
 
     public function execute()
     {
+        if (!$this->binaryName) {
+            $checker = new BaseSecurityChecker();
+        } else {
```

Comment:
I guess we should check also that `$this->binaryName === 'symfony'` because `binaryName` may be old security checker binary.

Reply:
Currently even `x` would work, because `symfony` [is checked anyway](#discussion_r394256566).
`bin/security-checker security:check --format=json` works, but `bin/security-checker security:check --format=json --dir=/xx/xy` will fail. I will see if the error message is understandable.